be/src/exec/pipeline/fragment_executor.cpp

```diff
@@ -127,34 +123,44 @@ Status FragmentExecutor::prepare(ExecEnv* exec_env, const TExecPlanFragmentParam
     }
 
     LOG(INFO) << "Prepare(): query_id=" << print_id(query_id)
-              << " fragment_instance_id=" << print_id(params.fragment_instance_id) << " backend_num=" << backend_num;
+              << " fragment_instance_id=" << print_id(params.fragment_instance_id)
+              << " backend_num=" << request.backend_num;
+
+    return fragment_ctx;
```

Comment:
why return fragment_ctx?

Reply:
Previously the `FragmentExecutor` stores a `FragmentContext*`, so must return a `shared_ptr<FragmentContext>` here.

But for some safety reason, I change it as `FragmentExecutor` stores a `shared_ptr<FragmentContext>`, so actually this function don't need to return a pointer.